ecs-autoscaling-cpu/autoscaling.tf

```diff
@@ -20,15 +15,15 @@ resource "aws_appautoscaling_target" "target" {
 }
 
 resource "aws_appautoscaling_policy" "up" {
-  name               = "${var.name_prefix}_ECS_ScaleUp"
+  name               = "${var.ecs_service_name}_ecs_scale_up"
   service_namespace  = "ecs"
   resource_id        = local.resource_id
   scalable_dimension = "ecs:service:DesiredCount"
 
   step_scaling_policy_configuration {
     adjustment_type         = "ChangeInCapacity"
     cooldown                = 60
-    metric_aggregation_type = var.metric_aggregation_type
+    metric_aggregation_type = "Average"
```

Comment:
A tohle uz teda nechcem configurovatelny? Jakoze realne neivm, jestli je case, kdy bysme tam chteli neco jinyho 🤔

Reply:
No ono to nebylo konfigurovatelný tady, takže by to stejně fungovalo jen tak napůl:
![CleanShot 2024-04-15 at 09 45 28](https://github.com/FigurePOS/terraform-modules/assets/215660/f3249da8-e795-4df2-9072-cdf5d0e54239)
A když jsem tak nad tím přemýšlel, tak mi přišlo že to asi ani není (zatím) třeba.(?)

Reply:
icic, tak cajk